apps/meteor/tests/end-to-end/api/livechat/contacts.ts

```diff
@@ -594,6 +594,65 @@ describe('LIVECHAT - contacts', () => {
 		});
 	});
 
+	describe('Contact Rooms', () => {
+		before(async () => {
+			await updatePermission('view-livechat-contact', ['admin']);
+		});
+
+		after(async () => {
+			await restorePermissionToRoles('view-livechat-contact');
+		});
+
+		it('should create a contact and assign it to the room', async () => {
+			const visitor = await createVisitor();
+			const room = await createLivechatRoom(visitor.token);
```

Comment:
What about using `let` (globally in this describe) and deleting the visitor and closing the room on an `afterEach` instead?
Not a big deal, though